src/components/workflow/DashboardFilters/SubmitterFilter.vue

```diff
@@ -0,0 +1,132 @@
+						-->
+					</div>
+				</div>
+			</div>
+			<button class="btn btn-info" type="button" @click="closeFilter()">
+            Done
+            </button>
+		</div>
+		</div>                   
+</template>
+
+<script>
+import { sync } from 'vuex-pathify'
+import WorkflowService from '../../../service/workflow-service';
+import VueBootstrapTypeahead from 'vue-bootstrap-typeahead';
+import _ from 'underscore';
+
+export default {
+  name: 'Workflow',
+  components:{
+	VueBootstrapTypeahead
+  },
+  data(){
+    return {
+		workflowService: new WorkflowService(),
+		visible : false,
+		searchValue: '',
+		submitterList:[],
+		filterSuccess:false,
+		selectedSubmitters : []
+    }
+  },
+  computed:{
+	workflowDashboard: sync("workflowDashboard"),
+	getSubmitters(){
+		var submitters=[];
+		var i=0;
+		for(i=0;i<this.workflowDashboard.rows.length;i++){
+			submitters.push(this.workflowDashboard.rows[i].submitter);
+		}
+		submitters = submitters.filter((item, i, ar) => ar.indexOf(item) === i);
+		console.log("unique submitters"+submitters);
+		return submitters;
+	},
+  },
+  props: {
+  },
+  methods:{
+    
+	addSubmitter($event){
+		//console.log("selected submitters is:"+this.searchValue+"  "+$event);
+		if(this.selectedSubmitters.length >0 ){
+			if(this.selectedSubmitters.indexOf(this.searchValue) == -1)
+				this.selectedSubmitters.push(this.searchValue);
+		}
+		else
+			this.selectedSubmitters.push(this.searchValue);
+		this.workflowDashboard.searchQuery.filterBySubmitters = this.selectedSubmitters;
+		console.log("selected submitters are:"+this.workflowDashboard.searchQuery.filterBySubmitters);
+	},
+	removeSubmitter(index){
+		console.log("removing index:"+index);
+		//this.selectedSubmitters.pop(this.selectedSubmitters[index]);
+		var removed = this.selectedSubmitters.splice(index,1);
+		console.log("selected submitters are:"+this.selectedSubmitters +" and removed element is:"+removed);
+	},
+	closeFilter(){
+		this.searchValue='';
+		this.visible=false;
+	}
+  },
+  
+  watch: {
```

Comment:
Are these watches necessary?  I assume you tested without them, but it feels like the job of the computed values is to watch dependent properties.  If we add a console.log to the getSubmitters, does it get called twice?